plugins/Morpheus/javascripts/ajaxHelper.js

```diff
@@ -74,6 +74,7 @@ function ajaxHelper() {
     /**
      * Should ajax request be asynchronous
      * @type {Boolean}
+     * @deprecated Synchronous XHR requests are deprecated(see http://xhr.spec.whatwg.org)
```

Comment:
We should definitely mention this in https://github.com/piwik/piwik/blob/master/CHANGELOG.md under "Deprecations". We should also mention that we will remove it in Piwik 3.0. I created follow up issue https://github.com/piwik/piwik/issues/8246